writer/template_test.go

```diff
@@ -0,0 +1,32 @@
+package writer
+
+import (
+	"os"
+	"text/template"
+)
+
+func ExampleTemplate_String() {
```

Comment:
**[govet]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
ExampleTemplate_String refers to unknown field or method: Template.String